### DISCUSSION TITLE
Umbraco Forms: Ability to create custom list view for entries

### DISCUSSION THREAD

**Discussion by beingonlinedave** [umbraco/Umbraco-CMS#13813] [Features and ideas]
The ability to choose which fields are displayed in the table of entries is a feature that has been asked for. This allows the user to quickly scan for important information without having to export and review in a spreadsheet.

**Comment by nul800sebastiaan** [comment]
Thanks @beingonlinedave - Umbraco Forms has a specific issue tracker where you can create feature requests here:  https://github.com/umbraco/Umbraco.Forms.Issues/issues/new?assignees=&labels=&template=2_Feature_request.md 

Make sure to leave your idea there instead! 👍
